tests/dio-randread.py

```diff
@@ -0,0 +1,28 @@
+import sys
+from PerfTest import FioTest
+from utils import get_fstype
+from utils import set_readpolicy
+from utils import get_active_readpolicy
+from utils import has_readpolicy
+
+class DioRandread(FioTest):
+    name = "diorandread"
+    command = ("--name diorandread --direct=1 --size=1g --rw=randread "
+               "--runtime=60 --iodepth=1024 --nrfiles=16 "
+               "--numjobs=16 --group_reporting")
+
+    def setup(self, config, section):
+        device = config.get(section, 'device')
+
+        if not get_fstype(device) == "btrfs":
+            return
+
+        if config.has_option(section, 'readpolicy'):
+            policy = config.get(section, 'readpolicy')
+            if not has_readpolicy(device):
+                print("Config:{} kernel does not support readpolicy", section)
+                sys.exit(1)
```

Comment:
I don't want to exit, that'll stop the tests completely.

We're running in a try/except block, maybe just add a NotRunException and raise that, and then handle it in the main run_test() function in fsperf.py.  That way the proper cleanup is done and we continue running the rest of the tests.

Reply:
Yeah, we need not run here. I will fix it.